TeamShiksha/logoexecutive
# Dsc feature fix new branch

### Summary
This PR updates the pricing section, navigation, and Home page layout:
- Replaced the dollar symbol (`$`) with the rupee symbol (`₹`) in pricing cards.
- Changed pricing card labels to `Free`, `Pro`, and `Custom`.
- Added a route in `App.jsx` to navigate to `/pricing` when the pricing button is clicked.
- Adjusted the Home page layout to place the pricing section below the features section.

---

### Approach
- Updated the `PricingCard.jsx` to reflect rupee symbols and new labels.
- Modified `App.jsx` to include a `/pricing` route.
- Edited the `Home.jsx` layout to rearrange sections.
- Verified all changes through manual testing and code review.

---

### How to Test the Changes
1. Open the application.
2. Verify that the pricing cards now show the rupee symbol (`₹`) and updated labels.
3. Click the pricing button in the header and confirm it redirects to `/pricing`.
4. On the Home page, check that the pricing section appears below the features section.

---

### Screenshots or Recordings
*Attach screenshots here, if applicable.*
![image](https://github.com/user-attachments/assets/24d73f22-e085-4277-8ef1-627e2f9cd32a)
![image](https://github.com/user-attachments/assets/fb9100ba-8033-4db4-88bc-eff94c9913bc)



---

### Checklist
